.gitignore

```diff
@@ -19,3 +19,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+<<<<<<< HEAD
```

Comment:
Oops, looks like we have a merge conflict not handled correctly.

Reply:
fixed